test/Twig/Tests/EnvironmentTest.php

```diff
@@ -180,13 +180,12 @@ public function testExtensionsAreNotInitializedWhenRenderingACompiledTemplate()
 
     public function testAutoReloadCacheMiss()
     {
-        $template_name = __FUNCTION__;
-        $template_content = __FUNCTION__;
+        $templateName = __FUNCTION__;
+        $templateContent = __FUNCTION__;
```

Comment:
renaming those variables should be done in a separate PR, it just increases the diff ... :(